### PR TITLE
Schedule workflow to run at 9 a.m. JST every Friday

### LINKED ISSUE #48: Scheduling workflow for updating Lint rules
It seems appropriate to schedule for every Friday morning since Dart is released every Wednesday and Flutter every Thursday.

https://github.com/yumemi-inc/flutter-yumemi-lints/blob/ff801b40774ba42cbbf02ffadab3946b6f9aded8/.github/workflows/update-lint-rules.yaml#L3-L5

- https://dart.dev/get-dart/archive
- https://docs.flutter.dev/release/archive

### PR DESCRIPTION
**Pull request by trm11tkr**
## Issue

- close #48 

## Overview (Required)

Schedule at 9 AM JST (0 AM UTC) every Friday

## Links
https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows#schedule

### CODE REVIEW

File: .github/workflows/update-contributors.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - 'main'
+  schedule:
+  # Schedule at 9 AM JST (0 AM UTC) every Friday
+  - cron: '0 0 * * 5'
```
Comment (trm11tkr) [inline .github/workflows/update-contributors.yaml:9]: GitHub Actions' scheduled workflows are configured to run on UTC time. Currently, it's not possible to set a timezone directly in the cron syntax.

FYI: 
https://github.com/orgs/community/discussions/13454

**Review by blendthink — APPROVED** [review]
Thank you for your work! LGTM!

**Review by Yamasaki-pan961 — CHANGES_REQUESTED** [review]
@trm11tkr 
Thank you for correcting! 
I think the issue #48 is a schedule for lint rule updates, but you have added a schedule to update-contributors.yaml.

File: .github/workflows/update-contributors.yaml (hunk 1 above)
Comment (morikann) [inline .github/workflows/update-contributors.yaml:9]: nits-badge 
Thanks for your work!
I think you should lower the indent of comment and cron. [FYI](https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows#schedule)

Comment (trm11tkr) [reply .github/workflows/update-contributors.yaml]: Thank you for pointing that out! I completely missed it.
I fixed [391cb19](https://github.com/yumemi-inc/flutter-yumemi-lints/pull/56/commits/391cb19a92bff58f8396bcde79ae1f5aadcd39b9)

**Comment by trm11tkr** [comment]
@Yamasaki-pan961 
> I think the issue https://github.com/yumemi-inc/flutter-yumemi-lints/issues/48 is a schedule for lint rule updates, but you have added a schedule to update-contributors.yaml.

Thank you for pointing that out! I've made the correction in the appropriate file.
[391cb19](https://github.com/yumemi-inc/flutter-yumemi-lints/pull/56/commits/391cb19a92bff58f8396bcde79ae1f5aadcd39b9)

File: .github/workflows/update-contributors.yaml (hunk 1 above)
Comment (morikann) [reply .github/workflows/update-contributors.yaml]: Thanks for the correction!

**Review by morikann — APPROVED** [review]
Thanks for your work! LGTM!

**Review by blendthink — APPROVED** [review]
LGTM!

**Review by Yamasaki-pan961 — APPROVED** [review]
LGTM!!

**Pull request merged**